<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAirlineTicketBooking</class>
 <widget class="QWidget" name="FrmAirlineTicketBooking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{border:1px solid rgb(100,100,100);}
#frame{border-radius:10px;}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(223, 237, 233,100);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>550</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{width:275}
QTabBar::tab{height:40}
</string>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1" native="true">
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>250</y>
          <width>69</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>人数</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_DepartDate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>182</y>
          <width>110</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>170</y>
          <width>69</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>出发日期</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>69</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>出发城市</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>260</y>
          <width>44</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>69</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>到达城市</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_SigleSearch">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>320</y>
          <width>93</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_Trans">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>60</y>
          <width>40</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resource.qrc">
          <normaloff>:/image/redir</normaloff>:/image/redir</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <widget class="QComboBox" name="DepartCity">
        <property name="geometry">
         <rect>
          <x>79</x>
          <y>130</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="ArriveCity">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>130</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
